lib/gerry/client/request.rb

```diff
@@ -65,7 +65,13 @@ def parse(response)
           raise_request_error(response)
         end
         if response.body
-          JSON.parse(remove_magic_prefix(response.body))
+          source = remove_magic_prefix(response.body)
```

Comment:
Refactoring suggestion here would be to make a method:

```
def jsonify(body)
...
```
that is responsible for all the JSON cleanup.